spec/ar2dto/has_dto_spec.rb

```diff
@@ -65,23 +67,82 @@
     end
 
     it "is not equal to another DTO of the same class with different attributes" do
-      other_user = User.new(attributes.merge(name: "Kent")).to_dto
+      other_user = User.new(attributes.merge(first_name: "Kent")).to_dto
 
       expect(subject).not_to eq(other_user)
     end
 
     it "is not possible to set values from outside" do
-      expect { subject.name = "Arturito" }.to raise_error(NoMethodError)
+      expect { subject.first_name = "Martin" }.to raise_error(NoMethodError)
+    end
+
+    context "when including methods" do
+      let(:options) do
+        { methods: [:full_name] }
+      end
+
+      it "becomes accessible in the DTO" do
+        expect(subject.full_name).to eq("Sandy Doe")
+      end
+    end
```

Comment:
Maybe add a test that is the opposite of this?
If the option for that method is not added, expect that the method is not availabe?